src/main/java/com/kodilla/ecommercee/dto/OrderDto.java

```diff
@@ -0,0 +1,20 @@
+package com.kodilla.ecommercee.dto;
+
+
+import com.kodilla.ecommercee.domain.Product;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@AllArgsConstructor
+@Getter
+public class OrderDto {
+
+    private Long id;
+    private LocalDate date;
+    private List<Product> productList;
```

Comment:
lista powinna przechowywać ProductDto.  
Jeżeli ukryjemy jakieś pole w ProductDto - powiedzmy że do Product ktoś doda pole SERIAL NUMBER do jakiegoś oprogramowania, celowo pomijamy to pole w ProductDto.     Jednak gdybyśmy zostawili twoje rozwiązanie        json  z OrderDto odkrył by wszystkie dane zanim zamówienie zostało by opłacone.